src/zap/zapinfo.cpp

```diff
@@ -2131,7 +2131,10 @@ void ZapInfo::getCallInfo(CORINFO_RESOLVED_TOKEN * pResolvedToken,
         if (pResult->thisTransform == CORINFO_BOX_THIS)
         {
             // READYTORUN: FUTURE: Optionally create boxing stub at runtime
-            m_zapper->Warning(W("ReadyToRun: Implicit boxing for calls to constrained methods not supported\n"));
+            if (doCompilationWarnings())
```

Comment:
> If we delete this warning, that would mean to me, that we don't think that we should ever address this

The GitHub issues are the best way to track work items ... adding a comment line switch won't help us to track it.

If you would like to still have a way to see this warning somehow, you can change it to `CORZAP_LOGLEVEL_INFO` so that it gets printed with verbose output only. I just do not think it is worth it to have a new special command line switch for it.